lenovo/thinkpad/t14s/amd/gen4/default.nix

```diff
@@ -0,0 +1,9 @@
+{ lib, pkgs, ... }:
```

Comment:
Can you add this model to flake.nix and the table in the README?

Reply:
Done!